assembl/views/api2/auth.py

```diff
@@ -466,6 +466,11 @@ def assembl_register_user(request):
                 "We already have a user with this username.")),
                 ErrorTypes.EXISTING_USERNAME,
                 HTTPConflict.code)
+        if len(username) > 20:
+            errors.add_error(localizer.translate(_(
+                "The username must be less than 20 characters.")),
+                ErrorTypes.INVALID_USERNAME,
+                HTTPConflict.code)
```

Comment:
Nah, it's just a bad request code
https://httpstatuses.com/400
https://docs.pylonsproject.org/projects/pyramid/en/latest/api/httpexceptions.html